### PR TITLE
Update ArcGIS Toolbox metadata

### PR DESCRIPTION
**Pull request by hjkristenson**
This PR updates the metadata for the RGB Decomposition tool in the ArcGIS Toolbox to more accurately reflect tool requirements and behavior. It also cleans up some formatting and typos in all of the metadata xmls.

### CODE REVIEW

**Comment by hjkristenson** [comment]
To vet these changes, you need to view the toolbox in ArcGIS software. Attached are PDFs of HTML renderings of the metadata for both the RGB Decomp and Scale Conversion tools (both of which underwent cleaning). The formatting (particularly spacing) is a bit wonky, but looks great when viewing the help within the tool dialog. 
[RGB_Decomposition_Tool_XML.pdf](https://github.com/ASFHyP3/asf-tools/files/10397163/RGB_Decomposition_Tool_XML.pdf)
[Scale_Conversion_Tool_XML.pdf](https://github.com/ASFHyP3/asf-tools/files/10397164/Scale_Conversion_Tool_XML.pdf)

**Comment by hjkristenson** [comment]
Here's an example of how the xml is displayed through the dialog. There's none of the weird spacing stuff. As I recall, I had to keep some of the spacing tags for it to render properly in ArcMap; I should probably check to make sure these hyperlink cleanups don't cause problems there....
SampleDialogHelp

**Comment by hjkristenson** [comment]
The versions with the ugly formatting are what you see if you click the help button in the tool interface. The rendering in the ArcGIS Pro Metadata tab is pretty good, though. I've attached PDFs of that view for each of the tools and the toolbox overall. Much easier to read through to validate content.
[LogDifferenceMetadata.pdf](https://github.com/ASFHyP3/asf-tools/files/10397458/LogDifferenceMetadata.pdf)
[ReclassifyRTCMetadata.pdf](https://github.com/ASFHyP3/asf-tools/files/10397459/ReclassifyRTCMetadata.pdf)
[RGBDecompositionMetadata.pdf](https://github.com/ASFHyP3/asf-tools/files/10397460/RGBDecompositionMetadata.pdf)
[ScaleConversionMetadata.pdf](https://github.com/ASFHyP3/asf-tools/files/10397461/ScaleConversionMetadata.pdf)
[ToolboxMetadata.pdf](https://github.com/ASFHyP3/asf-tools/files/10397462/ToolboxMetadata.pdf)
[UnzipMetadata.pdf](https://github.com/ASFHyP3/asf-tools/files/10397463/UnzipMetadata.pdf)

I also verified that the metadata renders just fine in ArcMap.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.6]
+
+### Changed
+* Updated the metadata for the RGB Decomposition tool in the ArcGIS Toolbox to more accurately reflect behavior
```
Comment (jtherrmann) [inline CHANGELOG.md:12]: Should we also call out "cleans up some formatting and typos in all of the metadata xmls"?

**Review by jtherrmann — APPROVED** [review]

**Review by jtherrmann — APPROVED** [review]

**Pull request merged**